packages/studio-be/src/core/dialog/skill-service.ts

```diff
@@ -21,7 +21,11 @@ export class SkillService {
     }
 
     // TODO change when studio is updated, since actual doesn't support catchall
-    return { flow: completeFlow, transitions: partialFlow.transitions }
+    return {
+      flow: completeFlow,
+      transitions: partialFlow.transitions,
+      ...(partialFlow.previewElements && { previewElements: partialFlow.previewElements })
```

Comment:
how is this serialized in the case if previewElements is null  or undefined ? 

`...undefined`

Reply:
With null or undefined 

````javascript
{
 flow: ...,
 transition:...
}

```` 

with previewElements defined

````javascript
{
 flow: ... ,
 transition: ... ,
 previewElements: ...
}

````